docs/rules/prefer-screen-queries.md

```diff
@@ -60,6 +60,11 @@ const { rerender, unmount, asFragment } = render(<Foo />);
 rerender(<Foo />);
 asFragment();
 unmount();
+
+// using baseElement
+const { getByText } = render(<Foo />), { baseElement: treeA }
+// using container
+const { getAllByText } = render(<Foo />), { baseElement: treeA }
```

Comment:
I guess by comment above this should be `container` instead of `baseElement`

Reply:
fixed